### ISSUE TITLE
Support a wildcard/"other" label

### ISSUE THREAD

**Issue by Turbo87** [lerna/lerna-changelog#74]
It might make sense in some projects to also include all the PRs in the changelog that don't have matching labels on them. We should probably support some sort of wildcard that groups all the PRs under it that don't have any of the other labels on them.

**Comment by hzoo** [comment]
Yeah that plus https://github.com/lerna/lerna-changelog/issues/10 is the same (non PR commit), and totally you might miss labeling and that would be bad if it's not in the changelog because of that
